sample-viewmodel/src/test/java/com/example/molecule/viewmodel/PupperPicsPresenterTest.kt

```diff
@@ -30,19 +30,23 @@ import org.junit.Assert.assertEquals
 import org.junit.Test
 
 class PupperPicsPresenterTest {
+
+  private val seed: Model = Model(
+    loading = true,
+    breeds = emptyList(),
+    currentBreed = null,
+    currentUrl = null,
+  )
+
   @Test
   fun `on launch, breeds are loaded followed by an image url`() = runBlocking {
     val picsService = FakePicsService()
+    val presenter = PupperPicsPresenter(picsService)
```

Comment:
This change can probably be reverted since we can still write the `PupperPicsPresenter` inline as:

```kotlin
moleculeFlow(mode = RecompositionMode.Immediate) {
    PupperPicsPresenter(seed, emptyFlow(), picsService)
}
```

Reply:
Working on it 👌

Reply:
Done